idp_data/idp_data/models.py

```diff
@@ -1,3 +1,38 @@
 from django.db import models
+from datetime import date
+from ckeditor.fields import RichTextField
 
-# Create your models here.
+
+class Category(models.Model):
+    name = models.CharField(max_length=500)
+    icon = models.CharField(max_length=50, default='')
+
+    def __str__(self):
+        return self.name
+
+
+class Municipality(models.Model):
+    code = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.code
+
+
+class Event(models.Model):
+    muni = models.ForeignKey(Municipality, on_delete=models.DO_NOTHING, default=None, blank=True, null=True)
+    title = models.CharField(max_length=500)
+    short_desc = models.CharField(max_length=200)
+    category = models.ForeignKey(Category, on_delete=models.DO_NOTHING)
+    confirmed_date = models.DateField("date added", default=date.today)
```

Comment:
This should be optional - see https://openupsa.slack.com/archives/CUGG4GTHQ/p1597854126015200